spec/features/admin/admin_user_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'feature_helper'
+
+describe 'ADMIN: AdminUsers' do
+  before { sign_in_as_admin }
+
+  let(:business) { create(:business_complete ) }
+  let(:user) { create(:user, business: business) }
+
+  it 'Creates an admin user' do
+    click_link 'Admin Users'
+    click_on 'New Admin User'
+    fill_in 'Email', with: 'test@example.com'
+    fill_in 'Password', with: 'password'
+    click_on 'Create Admin user'
+    expect(page).to have_content('Admin user was successfully created.')
+  end
+
+  it 'Edits an admin user' do
+    create(:admin_user, email: 'test@example.com', password: 'password')
+    click_link 'Admin Users'
+    click_link 'test@example.com'
+    click_on 'Edit Admin User'
+    fill_in 'Password', with: 'password'
+    click_on 'Update Admin user'
+    expect(page).to have_content('Admin user was successfully updated.')
+  end
+
+  @javascript
```

Comment:
The only test you need is that the edit page loads. take out everything else. 
The code to create/edit/delete is inside a gem activeadmin, so it is not ours to test.
What we got wrong here was the link, so just test that. You'll see a similar pattern in the rest of the admin specs: just enough to see the page load.